### DISCUSSION TITLE
How to disable Delete Account for Jetstream

### DISCUSSION THREAD

**Discussion by lucyacat** [laravel/framework#34730] [Q&A] [answered]
Hello, I tried Jetstream, it's really awesome (seems a perfect Laravel UI alternative). But I have one small issue, what is the best practice to disable Delete Account section?

Thanks.

**Comment by lucyacat** [comment, marked as answer]
Simply comment it in the published blade file

**Reply by ecdesa** [reply]
just because you commented it out in the UI doesnt mean the routes stop working.

**Reply by joelbutcher** [reply]
@mohamadnorouzi20 @lucyacat  Jetstream V2 will introduce a new `accountDeletion` feature option. If this option isn't present in your `jetstream.php` config file, then Livewire form won't be present, and the Inertia route won't be available

**Comment by AdvancedStyle** [comment]
How would you go about safely disabling this?  It seems like by deleting the component users could manually send a request to the route endpoint and delete their account (if they were technically minded)

**Reply by akifrabbani** [reply]
You can comment out everything inside delete() function in App\Actions\Jetstream\DeleteUser so it will not process the delete logic.

**Comment by techinfocomp** [comment]
Anyone finds a way to resolve this issue ?

**Comment by joelbutcher** [comment]
A solution to this discussion will be introduced in Jetstream V2. It'll add a new `accountDeletion` feature option. If this option isn't present in your `jetstream.php` config file, then the Livewire form won't be present, and the Inertia route won't be available.

**Reply by FoksVHox** [reply]
I know this is offtopic, but when will you release your package from #444?

**Reply by joelbutcher** [reply]
@FoksVHox Well, I've not started extracting it yet, as I also have a full-time job I need to dedicate 7 hours a day too... I'll probably look at this once Jetstream V2 is released and us that as the minimum version

**Reply by joelbutcher** [reply]
@FoksVHox Just issued the first pre-release version v0.0.1 of this package [here](https://github.com/joelbutcher/socialstream).

I intend to ship v1 on Monday and work on v2 for Jetstream v2 after that. If you have any issues, hit me up in Discord or add the issue to the repo

**Reply by FoksVHox** [reply]
@joelbutcher Lovely

**Reply by saeed-khodkari** [reply]
hi, special thanks to you

**Comment by JosueSoto23** [comment]
What is the route to delet users?
